match schedule view doesn't mention the day

In a CMP match schedule like this:
![match-schedule](https://cloud.githubusercontent.com/assets/1043120/7292301/a1d600e0-e94f-11e4-9a9d-4205105acd3c.png)
you have to look closely to notice that the match times jump backwards, no doubt across a day boundary.

It'd be nice to put a visual indicator (bold line or color change or something) at the day boundary (an overnight gap in the match schedule) and also show the day of week next to the match time (in the viewer's potentially distant time zone) whenever it differs from the preceding entry.

Can anyone else build the current 2015-comp-freeze code?

Android Studio 1.1.0 has 16 build errors in MatchTest.java trying to access static Match methods:

> C:\sourcecontrolled\the-blue-alliance-android\android\src\test\java\com\thebluealliance\androidclient\test\models\MatchTest.java:47: error: cannot find symbol
        JsonObject redAlliance = Match.getRedAlliance(alliances);
                                      ^
  symbol:   method getRedAlliance(JsonObject)
  location: class Match
...

But the IDE's control-click-through works fine on these symbols. Also, EventHelperTest builds OK. These unit test build errors keep the app from running on a device.

AS also doesn't like this line in android/build.gradle
> compile 'com.google.http-client:google-http-client-gson:1.19.0' exclude module: 'httpclient'

complaining `Cannot resolve symbol 'compile'`

The current AS version, 1.2 RC2, doesn't fare any better.

--- a/android/src/main/java/com/thebluealliance/androidclient/views/MatchView.java
+++ b/android/src/main/java/com/thebluealliance/androidclient/views/MatchView.java
@@ -1,5 +1,6 @@
 package com.thebluealliance.androidclient.views;
 
+import android.annotation.SuppressLint;
 import android.content.Context;
 import android.graphics.Typeface;
 import android.text.format.DateFormat;
@@ -16,6 +17,7 @@ import com.thebluealliance.androidclient.helpers.MatchHelper;
 import com.thebluealliance.androidclient.listeners.MatchClickListener;
 import com.thebluealliance.androidclient.listeners.TeamAtEventClickListener;
 
+import java.text.SimpleDateFormat;
 import java.util.Date;
 
 /**
@@ -231,9 +233,14 @@ public class MatchView extends FrameLayout {
                 // Match has no time
                 localTimeString = getContext().getString(R.string.no_time_available);
             } else {
+                // Format the day-of-week & time in the current locale with the user's 12/24-hour
+                // preference. The day part distinguishes today's matches from tomorrow's matches
+                // and from yesterday's matches with delayed results.
                 Date date = new Date(time * 1000L);
                 java.text.DateFormat format = DateFormat.getTimeFormat(getContext());
-                localTimeString = format.format(date);
+                @SuppressLint("SimpleDateFormat")
+                java.text.DateFormat dowFormat = new SimpleDateFormat("E ");
+                localTimeString = dowFormat.format(date) + format.format(date);
             }
 
             this.time.setText(localTimeString);

